flowplayer.hlsjs.js

```diff
@@ -602,7 +602,20 @@
                                             maxLevel = hls.loadLevel;
                                         }
                                         break;
-
+                                    case "FRAG_PARSING_METADATA":
+                                        data.samples.forEach(function(sample) {
+                                          var metadataHandler = function() {
+                                            if (videoTag.currentTime < sample.dts) return;
+                                            videoTag.removeEventListener('timeupdate', metadataHandler);
+                                            var raw = new TextDecoder('utf-8').decode(sample.data);
```

Comment:
That polyfill is huge, it can't be baked inside this engine, solely for ID3 support. Some other method has to be found or then it must be noted somewhere that ID3 support needs that polyfill for IE-support.